lib/tasks/data_task_spouse_dependent_ssns.rake

```diff
@@ -0,0 +1,43 @@
+require 'csv'
+
+namespace :data_task_spouse_dependent_ssns do
+  desc "create file of unhashed SSNs and respective unhashed spouse and dependent SSNs"
+  task create_file: :environment do
+    file_in = File.open("#{Rails.root}/tmp/ca_ids_for_eng_nov02.csv")
+    in_data = CSV.read(file_in, headers: true)
+    client_ids = []
+    in_data.each do |record|
+      record = record.to_ary
+      client_id = record[0][1]&.strip
+      client_ids << client_id.to_i
+    end
+
+    temp_file = Tempfile.open(%w[temp_file .csv], "tmp/")
+    CSV.open(Rails.root.join(temp_file.path), 'wb') do |csv|
+      csv << %w{client_id ssn type}
+
+      client_ids.each do |id|
```

Comment:
start with `Client.where(id: client_ids).includes(:intake, :dependents)` and each over that, so we don't have to fetch them one-by-one

maybe do .find_in_batches